src/components/dev-dot-content/mdx-components/mdx-table/mdx-table.module.css

```diff
@@ -3,72 +3,61 @@ We are using a wrapper here to enable horizontal scrolling while still retaining
 
 TODO: implement a more accessible horizontal overflow solution as KB-only users aren't able to focus and scroll these elements
 */
-.tableWrapper {
-  margin-bottom: 20px;
-  max-width: 100%;
-  overflow-x: auto;
-  width: 100%;
-
-  & table {
-    background-color: transparent;
-    border-collapse: collapse;
-    border: 1px solid #ddd;
-    border-right: 0;
-    border-spacing: 0;
-    max-width: 100%;
-    width: 100%;
-
-    & th {
-      text-align: left;
-    }
-
-    & td,
-    & th {
-      border-right: 1px solid #ddd;
-      padding: 0;
-    }
-
-    & > tbody > tr:nth-of-type(odd) {
-      background-color: #f9f9f9;
-    }
-
-    & > thead > tr > th,
-    & > tbody > tr > th,
-    & > tfoot > tr > th,
-    & > thead > tr > td,
-    & > tbody > tr > td,
-    & > tfoot > tr > td {
-      padding: 8px;
-      line-height: 1.42857143;
-      vertical-align: top;
-      border-top: 1px solid #ddd;
-    }
 
-    & > thead > tr > th {
-      vertical-align: bottom;
-      border-bottom: 1px solid #ddd;
-    }
-
-    & > caption + thead > tr:first-child > th,
-    & > colgroup + thead > tr:first-child > th,
-    & > thead:first-child > tr:first-child > th,
-    & > caption + thead > tr:first-child > td,
-    & > colgroup + thead > tr:first-child > td,
-    & > thead:first-child > tr:first-child > td {
-      border-top: 0;
-      border-right: 1px solid #ddd;
-    }
-
-    & > tbody + tbody {
-      border-top: 1px solid #ddd;
-    }
-
-    & table {
-      background-color: #fff;
-    }
-
-    & tr.warning td {
-      background: #fcf8e3;
-    }
-  }
+.tableWrapper {
```

Comment:
Thoughts on adding the border to the wrapper to ensure the outer border is always visible, even when the table scrolls? On the right you can see that the wrapping border is visible vs the border being hidden?

<img width="1806" alt="CleanShot 2022-09-15 at 09 53 39@2x" src="https://user-images.githubusercontent.com/825855/190422763-c8556e73-42c2-4111-9d06-00bf4a935c75.png">

Accompanying CSS:

```css
.tableWrapper {
	--background-color: #fafafa;
	--border-color: rgba(101, 106, 118, 0.2);
	--border-radius: 6px;

	border: 1px solid var(--border-color);
	overflow: hidden;
	border-radius: var(--border-radius);
	width: fit-content;
	max-width: 100%;
	overflow-x: auto;

	& table {
		border-collapse: collapse;
		border-style: hidden;
	}

	& thead {
		background-color: var(--background-color);
	}

	& tbody tr:nth-child(even) {
		background-color: var(--background-color);
	}

	& th,
	& td {
		border: 1px solid var(--border-color);
		padding: 8px 12px;
	}
}

```

Is there accompanying HDS variables we should be using for the border and background colors here?

Reply:
I think I prefer the one on the left.

Reply:
I think I prefer the one on the left.